MessengerProj/src/main/res/values/strings.xml

```diff
@@ -204,23 +204,23 @@
     <!--permissions-->
     <string name="PermissionContacts">Access to contacts needed to connect with your friends across all your devices.</string>
     <string name="PermissionStorage">Storage access needed to send and save photos, videos, music, and other media.</string>
-    <string name="PermissionNoAudio">Microphone access needed to send voice messages.</string>
+    <string name="PermissionNoAudio">Mic access needed to send voice messages.</string>
     <string name="PermissionOpenSettings">Settings</string>
     <!--Intro view-->
     <string name="Intro1Headline">Delta Chat</string>
-    <string name="Intro1Message">The messenger with the <![CDATA[<b>broadest audience</b>]]> in the world. <![CDATA[<b>Free</b>]]> and <![CDATA[<b>independent</b>]]>.</string>
+    <string name="Intro1Message">The messenger with the <![CDATA[<b>broadest reach</b>]]> worldwide. <![CDATA[<b>Free</b>]]> and <![CDATA[<b>independent</b>]]>.</string>
 
     <string name="Intro2Headline">Independent</string>
-    <string name="Intro2Message"><![CDATA[<b>There is no Delta Chat server.</b>]]> The app only uses your email server.</string>
+    <string name="Intro2Message"><![CDATA[<b>There is no Delta Chat server.</b>]]> The app only uses your mail server.</string>
 
     <string name="Intro3Headline">Fast</string>
-    <string name="Intro3Message"><![CDATA[<b>Push messages</b>]]> in seconds.<![CDATA[<br/>]]>Rapid interface.</string>
+    <string name="Intro3Message"><![CDATA[<b>Delivery</b>]]> in seconds.<![CDATA[<br/>]]>Rapid interface.</string>
 
     <string name="Intro4Headline">Powerful</string>
     <string name="Intro4Message"><![CDATA[<b>Unlimited</b>]]> chats, images, videos, audio messages and more. Multi-client capable.</string>
 
     <string name="Intro5Headline">Free</string>
-    <string name="Intro5Message"><![CDATA[<b>Delta Chat</b>]]> is free forever.<![CDATA[<br/>]]>Open-source. No ads. No subscription. No vendor lock-in.</string>
+    <string name="Intro5Message"><![CDATA[<b>Delta Chat</b>]]> is forever gratis.<![CDATA[<br/>]]>Libre copyleft. No ads. No subscription. No vendor lock-in.</string>
```

Comment:
This is not the same. I think the "free forever" mainly means "freedom" here, see https://en.wikipedia.org/wiki/Gratis_versus_libre

Reply:
Me thinking you think you are right is the only concept i am certain about there.
That idea was also conveyed that way to me, which tells me something about me. What that same reasoning tells me it tells someone else, is any variation thereof.
The precision with which the intended concept of "free" and "open-source" is stated, (especially in that order,) allows for meaningless interpretation.

So, promising it to always to be gratis is then actually a new claim, and the connotation between that and the rest is troublesome.
How about: 
> "Delta chat **is copyleft**, forever libre. No ads …